### PR TITLE
New features

### PR DESCRIPTION
**Pull request by Kiruba2602**

### CODE REVIEW

**Comment by Kiruba2602** [comment]
1. Removed unwanted json object from Clients-appointment, Client-bill, Client-notes.
2. Added Appointmentform in Appointment page.
3. Removed the unwanted header added by another person in Billingdetails

**Comment by svb-hikip** [comment]
Hey @Kiruba2602  Good work buddy! There are a few updates which I had done yesterday night, can you check if there is any conflict?

**Comment by Kiruba2602** [comment]
Yes, I noted that and I added a addappointmentform in src/components/appointments and a pop-up which show the appointment details in appointment tab instead of using alert.

File: src/index.css
Comment (svb-hikip) [inline src/index.css:1]: Loader already added by @anthonym995 Is this required?

File: src/pages/practice/BillingDetails.jsx
Comment (svb-hikip) [inline src/pages/practice/BillingDetails.jsx:1]: Why remove the header? is it not looking good? What are your thoughts?

File: src/index.css
Comment (Kiruba2602) [reply src/index.css:1]: But It isn't added to the branch right.

File: src/pages/practice/AppointmentDetail.jsx
[the remark this answers is not in the document]
Comment (Kiruba2602) [reply src/pages/practice/AppointmentDetail.jsx:1]: Yes, I also have that idea glad you asked

File: src/pages/practice/BillingDetails.jsx
Comment (Kiruba2602) [reply src/pages/practice/BillingDetails.jsx:1]: In the header part of billing component has replica of tailwind & notification icon and a user profile with a dropdown which is already present but other added again over it that's why I removed it.

**Comment by Kiruba2602** [comment]
added suspense like the same added in clientsliststack by @anthonym995  and removed previously added changes in clientsliststack component.

**Pull request closed without merging**